cmd/flightctl-server/main.go

```diff
@@ -44,11 +45,17 @@ func main() {
 	if err != nil {
 		log.Fatalf("ensuring server cert: %v", err)
 	}
-	_, _, err = ca.EnsureClientCertificate(certFile(clientBootstrapCertName), keyFile(clientBootstrapCertName), clientBootstrapCertName, clientBootStrapValidityDays)
+	clientCert, _, err := ca.EnsureClientCertificate(certFile(clientBootstrapCertName), keyFile(clientBootstrapCertName), clientBootstrapCertName, clientBootStrapValidityDays)
```

Comment:
Would it make sense to move  keyFile etc funcs into the config? I would expect an explicit path passed by the user to the flag or a generated default using these funcs. Seems like once we are in server, agent etc the paths should be set by config already.

Reply:
Makes sense, yes. I'll create a separate PR that'll allow to override the default config path and makes key and cert paths part of the config object. Would a fixed path relative to the config path be ok or do we need them to be configurable, too?